ls1intum/Artemis
# `Programming exercises`: Use a fixed version for the Haskell test image

### Checklist
#### General


#### Server
#### Changes affecting Programming Exercises

### Motivation and Context
The Haskell exercises need to use a version of Stack that matches the one already contained in the Docker image to avoid rebuilding the complete GHC for every submission.

### Description
This sets a fixed tag for the Docker image so that the Stack version in the Docker image does not accidentally change.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a new Haskell programming exercise.
2. Check that the solution passes all tests, and the exercise passes no tests (⇒ the version update should not have broken the template).

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
unchanged
